File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/DXBindingModule.vb
Imports MVVMDemo.DXBindingDemo
Imports DevExpress.Xpf.DemoBase

Namespace MVVMDemo

    Public Partial Class DXBindingModule
        Inherits MVVMDemoModule

        Protected Overrides Function CreateShowcases() As IShowcaseInfo()
            Const path As String = "Modules/DXBinding"
            Const uri As String = "115771/MVVM-Framework/Productivity/DXBinding"
            Return New ShowcaseInfo() {LoadShowcase("Binding Expressions", uri, path, {GetType(BindingExpressionsView), GetType(BindingExpressionsViewModel)}), LoadShowcase("Functions", uri, path, {GetType(FunctionsView), GetType(FunctionsViewModel), GetType(RegistrationHelper)}), LoadShowcase("Binding Sources", uri, path, {GetType(SourcesView)}), LoadShowcase("Back Conversion", uri, path, {GetType(BackConversionView), GetType(BackConversionViewModel)})}
        End Function
    End Class
End Namespace

File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/DXBindingModule.cs
using MVVMDemo.DXBindingDemo;
using DevExpress.Xpf.DemoBase;

namespace MVVMDemo {
    public partial class DXBindingModule : MVVMDemoModule {
        protected override IShowcaseInfo[] CreateShowcases() {
            const string path = "Modules/DXBinding";
            const string uri = "115771/MVVM-Framework/Productivity/DXBinding";
            return new ShowcaseInfo[] {
                LoadShowcase("Binding Expressions", uri, 
                    path, new[] { typeof(BindingExpressionsView),  typeof(BindingExpressionsViewModel) }),
                LoadShowcase("Functions", uri,
                    path, new[] { typeof(FunctionsView),  typeof(FunctionsViewModel), typeof(RegistrationHelper) }),
                LoadShowcase("Binding Sources", uri,
                    path, new[] { typeof(SourcesView) }),
                LoadShowcase("Back Conversion", uri,
                    path, new[] { typeof(BackConversionView),  typeof(BackConversionViewModel) }),

            };
        }
    }
}
